client/src/styles/styles.css

```diff
@@ -78,3 +78,32 @@ body {
 .navigationlabel {
   @apply text-nl font-normal leading-nl;
 }
+
+/* Card Container */
+.card-container {
```

Comment:
@HuxleyBerry @SodaVolcano  I am not sure about if it is all good to write all the style in styles.css file